hieradata/role/webserver.yaml

```diff
@@ -16,62 +14,26 @@ psick::packages::packages_list:
   - "git"
   - "googlechrome"
   - "notepadplusplus.install"
+  - "unzip"
+  - "urlrewrite"
   - "webdeploy"
 
-# Useful registry keys: http://www.jasinskionline.com/TechnicalWiki/Useful-Windows-Registry-Keys.ashx?AspxAutoDetectCookieSupport=1
-# Windows registry: http://www.forensicswiki.org/wiki/Windows_Registry
-# Registry Guide for Windows: http://www.pctools.com/guides/registry/
-#
-psick::windows::registry::use_defaults: true
-
-psick::puppet::pe_agent::manage_environment: true
-psick::windows::features::use_defaults: true
 psick::windows::features::install:
   'Web-WebServer':
     installsubfeatures: true
-  'file-services':
```

Comment:
I think we need the file services to share files on SMB

Reply:
I'm [not sure that we do need it](http://geekswithblogs.net/Wchrabaszcz/archive/2013/09/04/how-to-install-windows-server-features-using-powershell--server.aspx), but anyway I moved it to `common.yaml`.